lane.py

```diff
@@ -1157,7 +1158,6 @@ def search(self, query, search_client, pagination=None):
                         lazyload(mw.license_pool, LicensePool.identifier),
                         lazyload(mw.license_pool, LicensePool.presentation_edition),
                     )
-                    q = self.only_show_ready_deliverable_works(q, mw)
```

Comment:
Are we going to build the featured lanes from the search index, or will that still use the database? Featured lanes aren't paginated, so I think it's really important that they only include links the default client can fulfill. I'm okay with paginated lists including works with no delivery mechanism fulfillable by the default client, assuming that our clients know enough to hide those entries. We'll need to test that before we deploy to production.